cpp/data/krimdok-beacon.header

```diff
@@ -3,6 +3,6 @@
 #TARGET: https://krimdok.uni-tuebingen.de/Search/Results?lookfor={ID}&Type=GND
 #NAME: KRIMDOK
 #MESSAGE: KRIMDOK: Publikationen der Kriminologie
-#DESCRIPTION: Die Datenbank "Krimdok" (IxTheo: https://krimdok.uni-tuebingen.de) ist eine internationale, open access Bibliographie für die Kriminologie. Ausgewertet werden Monographien (Bücher) und Aufsätze aus kriminologischen Zeitschriften sowie Aufsätze aus Sammelschriften (z.B. Kongressveröffentlichungen). Nachgewiesen werden auch einschlägige Datenbanken und Internetlinks. Statistik: ca 270.000 Mio Titel (Stand Oktober 2021).
+#DESCRIPTION: Die Datenbank "KrimDok" (https://krimdok.uni-tuebingen.de/) ist eine internationale, open access Bibliographie für Kriminologie. Ausgewertet werden alle Mediengattungen wie Monographien, Aufsätze (aus mehr als 200 kriminologischen und bezugswissenschaftlichen Zeitschriften sowie Aufsätze aus Sammelschriften, z.B. Festschriften und Kongressschriften), Rezensionen, Websites, Statistiken, Forschungsdaten etc. Derzeit beinhaltet KrimDok rund 300.000 kriminologische (Stand November 2022).
```

Comment:
The text seems to match the text in the issue, however the last sentence reads a bit strange so i'm gonna ask in the issue if it's correct.